update `ignore_annotations/labels` doc to include context for regexp in nested metadata objects (#2454)
hashicorp/terraform-provider-kubernetes: website/docs/index.html.markdown

Both attributes support RegExp to match metadata objects more effectively.

> ~> **Note:** RegExp will only work on root metadata objects. This makes using RegExp for ignoring annotations/labels in metadata objects that are nested such as `spec.template.metadata[0].annotations["..."]` not possible.

Reviewer: ```suggestion
~> **Note:** RegExp will only work on root metadata objects. It's currently not possible to use RegExp for ignoring annotations/labels in metadata objects that are nested such as `spec.template.metadata[0].annotations["..."]`.
```

Author: thank you for the suggestion!
